src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -120,6 +120,41 @@ class OMPSerialDirective(OMPDirective):
 
     '''
 
+    def validate_global_constraints(self):
+        '''
+        Perform validation checks that can only be done at code-generation
+        time.
+
+        :raises GenerationError: if this OMPSerial is not enclosed \
+                            within some OpenMP parallel region.
```

Comment:
Picky but since there are other comments, can you also indent this line to match the "if this", also in the next raises.

Reply:
Done